resources/migrations/000_migrations.yaml

```diff
@@ -14215,45 +14188,6 @@ databaseChangeLog:
             columnName: updated_at
             newDataType: DATETIME
 
-  - changeSet:
```

Comment:
I've collapsed these three migrations into one new one which adds the FK constraint with the onDelete policy. Didn't really need to have them as three in the first place (I think it was just due to the sequencing of work)